v2/emailpassword/email-delivery/smtp/configure-smtp.mdx

```diff
@@ -118,6 +119,7 @@ init(
                             name="...",
                             email="..."
                         ),
+                        username="...", # this is optional. In case not given, from.email will be used instead
```

Comment:
this is for the python SDK, which doesn't have this setting yet. Can you please remove this?